test/accessor_tests.cc

```diff
@@ -507,5 +507,13 @@ namespace detail {
 		});
 	}
 
+#if CELERITY_DETAIL_HAS_NO_UNIQUE_ADDRESS
```

Comment:
Consider moving this into the test and conditionally `SKIP`!

Reply:
Also done!